src/main/java/frc/robot/subsystems/Ramp.java

```diff
@@ -38,35 +38,23 @@ public void periodic() {
             SmartShuffleboard.put("Ramp", "Desired pos", rampPos);
             SmartShuffleboard.put("Ramp", "Reverse Switch Tripped", getReversedSwitchState());
             SmartShuffleboard.put("Ramp", "Forward Switch Tripped", getForwardSwitchState());
-//            double pidP = SmartShuffleboard.getDouble("Ramp", "PID P", neoPidMotor.getPidController().getP());
-//            double pidI = SmartShuffleboard.getDouble("Ramp", "PID I", neoPidMotor.getPidController().getI());
-//            double pidD = SmartShuffleboard.getDouble("Ramp", "PID D", neoPidMotor.getPidController().getD());
-//            double pidFF = SmartShuffleboard.getDouble("Ramp", "PID FF", neoPidMotor.getPidController().getFF());
-//            if (pidP != neoPidMotor.getPidController().getP()) neoPidMotor.getPidController().setP(pidP);
-//            if (pidI != neoPidMotor.getPidController().getI()) neoPidMotor.getPidController().setI(pidI);
-//            if (pidD != neoPidMotor.getPidController().getD()) neoPidMotor.getPidController().setD(pidD);
-//            if (pidFF != neoPidMotor.getPidController().getFF()) neoPidMotor.getPidController().setFF(pidFF);
-            SmartShuffleboard.put("Ramp", "Forward Switch Tripped", getForwardSwitchState());
-            SmartShuffleboard.put("Ramp", "Forward Switch Tripped", getForwardSwitchState());
-
+            SmartShuffleboard.put("Driver", "Speaker Close", isShootCloseAngle())
+                    .withPosition(9, 0)
+                    .withSize(1, 1);
+            SmartShuffleboard.put("Driver", "Speaker Away", isShootAwayAngle())
+                    .withPosition(9, 1)
+                    .withSize(1, 1);
+            SmartShuffleboard.put("Driver", "Amp", isShootAmpAngle())
+                    .withPosition(8, 1)
+                    .withSize(1, 1);
         }
 
         Logger.logDouble(baseLogName + "EncoderValue", getRampPos(), Constants.ENABLE_LOGGING);
```

Comment:
Move to if(debug)?

Reply:
done

Reply:
Still some logs left outside the condition

Reply:
done